core/src/main/java/com/google/errorprone/ErrorProneCompiler.java

```diff
@@ -142,7 +152,7 @@ public Builder report(ScannerSupplier scannerSupplier) {
     }
   }
 
-  public Result compile(String[] args) {
+  public Result run(String[] args) {
```

Comment:
Can you also rename the other instance methods named `compile`? The only usages are in `CompilationTestHelper` and `ErrorProneTestCompiler`.